aliroot.sh

```diff
@@ -49,6 +50,7 @@ cmake $SOURCEDIR                                                     \
       ${FASTJET_ROOT:+-DFASTJET="$FASTJET_ROOT"}                     \
       ${DPMJET_ROOT:+-DDPMJET="$DPMJET_ROOT"}                        \
       ${ZEROMQ_ROOT:+-DZEROMQ=$ZEROMQ_ROOT}                          \
+      ${ALITPCCOMMON_ROOT:+-DALITPCCOMMON_DIR=$ALITPCCOMMON_ROOT}    \
```

Comment:
Here I am assuming that the CMake variable `ALITPCCOMMON_DIR` will be properly dealt with in a separate AliRoot PR, am I right?